src/layout/FooterLayout.vue

```diff
@@ -1,5 +1,40 @@
-<script setup></script>
+<script setup>
+import github_gray from '@/assets/icons/github_gray.svg';
+
+const members = [
+  { name: 'wldnjs990', href: 'https://github.com/wldnjs990' },
+  { name: 'yoo-nji', href: 'https://github.com/yoo-nji/yoo-nji' },
+  { name: 'kodingzz', href: 'https://github.com/kodingzz/kodingzz' },
+  { name: 'AAminha', href: 'https://github.com/AAminha/AAminha' },
+  { name: 'y-yeji', href: 'https://github.com/y-yeji' },
+  { name: 'SeungHyeon-web', href: 'https://github.com/SeungHyeon-web' },
+];
+</script>
 
 <template>
-  <div>푸터</div>
+  <footer class="w-full h-32 bg-gray-5 flex items-center justify-center">
+    <article class="w-full max-w-[486px] flex flex-col">
+      <a
+        href="https://github.com/HanRiverVue/HanRiver"
+        target="_blank"
+        class="flex gap-[10px] items-center justify-center mb-5"
+      >
```

Comment:
```suggestion
      <a
        href="https://github.com/HanRiverVue/HanRiver"
        target="_blank"
        rel="noopener noreferrer"
        class="flex gap-[10px] items-center justify-center mb-5"
      >
```
이것도 추가하면 좋을 것 같습니당!!